test/migrations/V9000__test_data.sql

```diff
@@ -14,3 +14,8 @@ DELETE FROM archive;
 INSERT INTO archive (case_id, workflow_run_ids_for_offsite_archive, unload_file_for_offsite_archive, files_copied_to_offsite_archive_staging_dir, commvault_backup_job_id, workflow_run_ids_for_vidarr_archival, unload_file_for_vidarr_archival, files_loaded_into_vidarr_archival, case_files_unloaded) VALUES
 ((SELECT id FROM cardea_case WHERE case_identifier = 'R11_TEST_1000_Xy_Z'), '{"vidarr:research/run/f77732c812aa134f61b3a7c11d1c4451cefe70e90e828a11345e8a0cd7704a0f", "vidarr:research/run/eeb4c43908e5df3dd4997dcc982c4c0d7285b51d7a800e501da06add9125faa7", "vidarr:research/run/e651c4aa01d506904bc8b89a411e948c24d43fc0e841486937f23d72eb7c4fae", "vidarr:research/run/de7b18bb97916885afbb7b085d61f00cfaa28793a8b7260b50c4d4ece3567216"}', '{"a_json_file": "sure is"}', ('2023-06-22 16:49:37-04'::timestamptz), 'CJ123', '{"vidarr:research/run/da0e6032ed08591ae684a015ad3c58867a47a65b6c61995e421fc417e2c438c1"}', '{"this_is": "a_load_slash_unload_file_for_vidarr_archival"}', ('2023-06-21 09:38:13-04'::timestamptz), ('2023-06-23 04:03:02'::timestamptz)),
 ((SELECT id FROM cardea_case WHERE case_identifier = 'R12_TEST_1212_Ab_C'), '{"vidarr:research/run/ac7fa728822f4b801f17cb6aa5d597e32642e6e36a2089fb3c947e4d7d679203", "vidarr:research/run/93d25e166f7ea82c6eb17e48e62e4134939fcaf75c9b9f953819797f4878d961", "vidarr:research/run/8b7fb588926f5efcdfe9824e867b91c93829c383f4177f87b92288b2f2554f8f", "vidarr:research/run/88065a62eeb0ad351f48ff9a3a32cdbdbe4bedf1e386603c1d4d8cfd46d5acd0", "vidarr:research/run/86b9da7b2b84174daef0f29beecf84453d8f987c46805121bf3109e0de48dc14"}', NULL, NULL, NULL, '{"vidarr:research/run/d8cc36839b0dd246af6940f175f089bf2007f5dcb01a00bc53a55526060250f9"}', NULL, NULL, NULL);
+
+DELETE FROM signoff;
+INSERT INTO signoff (case_identifier, qc_passed, username, signoff_step_name, deliverable_type, comment) VALUES
+('R11_TEST_1000_Xy_Z', TRUE, 'me', 'QC Gate 1', 'Clinical', 'qc test pass'),
```

Comment:
Your design docs state that the signoff_tep_name and deliverable_type will be controlled vocabulary - it's worth setting up the test data to use that same controlled vocabulary.